web/massastation/src/pages/Store/Install.tsx

```diff
@@ -80,25 +80,23 @@ function Install({
               <div className="mas-caption text-neutral mb-3">
                 {Intl.t('store.sidepanel.subtitle')}
               </div>
-              {isLoading ? (
-                <div className="mas-body2 mb-4 text-neutral">
-                  {Intl.t('store.loading')}
-                </div>
-              ) : (
-                <>
-                  <Input
-                    placeholder={Intl.t('store.sidepanel.placeholder')}
-                    name="url"
-                    value={url}
-                    onChange={(e) => handleChange(e)}
-                    error={error}
-                    customClass="bg-primary mb-3"
-                  />
-                  <Button type="submit" customClass="mt-3">
-                    {Intl.t('store.sidepanel.button')}
-                  </Button>
-                </>
-              )}
+              <>
+                <Input
+                  placeholder={Intl.t('store.sidepanel.placeholder')}
+                  name="url"
+                  value={url}
+                  onChange={(e) => handleChange(e)}
```

Comment:
```suggestion
                  onChange={handleChange}
```

You don't need to call `e` here. It's automatic in react 🤔